pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -50,12 +50,22 @@
 )
 
 try:
-    from pennylane_lightning.lightning_qubit_ops import backend_info
+    from pennylane_lightning.lightning_qubit_nb import backend_info
 
     LQ_CPP_BINARY_AVAILABLE = True
-except ImportError as ex:
-    warn(str(ex), UserWarning)
-    LQ_CPP_BINARY_AVAILABLE = False
+except ImportError:
+    try:
+        # Fall back to pybind11 version if nanobind version is not available
+        from pennylane_lightning.lightning_qubit_ops import backend_info
+
+        warn(
+            "Using legacy pybind11 bindings. Consider upgrading to nanobind version for better performance.",
```

Comment:
Could we raise RuntimeError instead of UserWarning so that we can capture such failures in CIs?